### ISSUE TITLE
Shell app does not start after installing latest spartacus libs on a new shell app

### ISSUE THREAD

**Issue by hackergil** [SAP/spartacus#11893]
**Happens on Branch 3.2.0-rc.1**

When installing Spartacus with the latest changes of the installation script (`scripts/install/run.sh install`), installation is successful. But when we try to start the csr app (`scripts/install/run.sh start`), we get the errors below:

It seems to be related to the user library and the CommandService (which seems to be resolving types from `dist`)

> Compiling @spartacus/product/variants/assets : es2015 as esm2015
Compiling @spartacus/product/variants/root : es2015 as esm2015
Compiling @spartacus/product/variants : es2015 as esm2015
Compiling @spartacus/qualtrics/root : es2015 as esm2015
Compiling @spartacus/qualtrics : es2015 as esm2015

chunk {} runtime.9f93c3884ce8821f95b8.js (runtime) 1.45 kB [entry] [rendered]
chunk {1} main.b9260cbe3c7794a6919c.js (main) 128 bytes [initial] [rendered]
chunk {2} polyfills.9d83a6555742c190e0f6.js (polyfills) 130 bytes [initial] [rendered]
chunk {3} styles.c806d94fdb80f8a5bad8.css (styles) 368 kB [initial] [rendered]
Date: 2021-04-08T18:14:30.342Z - Hash: fdd870407aac18bfaa05 - Time: 23938ms

ERROR in node_modules/@spartacus/user/account/core/facade/user-account.service.d.ts:10:33 - error TS2307: Cannot find module '../../../../../dist/core/spartacus-core' or its corresponding type declarations.

10     protected userQuery: import("../../../../../dist/core/spartacus-core").Query<User, []>;
                                   ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
node_modules/@spartacus/user/profile/core/facade/user-email.service.d.ts:11:37 - error TS2307: Cannot find module '../../../../../dist/core/spartacus-core' or its corresponding type declarations.

11     protected updateCommand: import("../../../../../dist/core/spartacus-core").Command<{
                                       ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
node_modules/@spartacus/user/profile/core/facade/user-password.service.d.ts:10:37 - error TS2307: Cannot find module '../../../../../dist/core/spartacus-core' or its corresponding type declarations.

10     protected updateCommand: import("../../../../../dist/core/spartacus-core").Command<{
                                       ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
node_modules/@spartacus/user/profile/core/facade/user-password.service.d.ts:14:36 - error TS2307: Cannot find module '../../../../../dist/core/spartacus-core' or its corresponding type declarations.

14     protected resetCommand: import("../../../../../dist/core/spartacus-core").Command<{
                                      ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
node_modules/@spartacus/user/profile/core/facade/user-password.service.d.ts:18:57 - error TS2307: Cannot find module '../../../../../dist/core/spartacus-core' or its corresponding type declarations.

18     protected requestForgotPasswordEmailCommand: import("../../../../../dist/core/spartacus-core").Command<{
                                                           ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
node_modules/@spartacus/user/profile/core/facade/user-profile.service.d.ts:15:37 - error TS2307: Cannot find module '../../../../../dist/core/spartacus-core' or its corresponding type declarations.

15     protected updateCommand: import("../../../../../dist/core/spartacus-core").Command<{
                                       ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
node_modules/@spartacus/user/profile/core/facade/user-profile.service.d.ts:18:36 - error TS2307: Cannot find module '../../../../../dist/core/spartacus-core' or its corresponding type declarations.

18     protected closeCommand: import("../../../../../dist/core/spartacus-core").Command<undefined, unknown>;
                                      ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
node_modules/@spartacus/user/profile/core/facade/user-profile.service.d.ts:19:34 - error TS2307: Cannot find module '../../../../../dist/core/spartacus-core' or its corresponding type declarations.

19     protected titleQuery: import("../../../../../dist/core/spartacus-core").Query<Title[], []>;
                                    ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
node_modules/@spartacus/user/profile/core/facade/user-register.service.d.ts:15:39 - error TS2307: Cannot find module '../../../../../dist/core/spartacus-core' or its corresponding type declarations.

15     protected registerCommand: import("../../../../../dist/core/spartacus-core").Command<{
                                         ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
node_modules/@spartacus/user/profile/core/facade/user-register.service.d.ts:18:44 - error TS2307: Cannot find module '../../../../../dist/core/spartacus-core' or its corresponding type declarations.

18     protected registerGuestCommand: import("../../../../../dist/core/spartacus-core").Command<{
                                              ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
node_modules/@spartacus/user/profile/components/update-profile/update-profile-component.service.d.ts:14:32 - error TS2307: Cannot find module '../../../../../dist/user/profile/root/spartacus-user-profile-root' or its corresponding type declarations.

14     titles$: Observable<import("../../../../../dist/user/profile/root/spartacus-user-profile-root").Title[]>;
                                  ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

**Comment by marlass** [comment]
Resolved on develop and on release/3.2.x branch

**Comment by KateChuen** [comment]
Tested on dev9. No more errors. Shell app starts. 
Tested on these branches
- develop
- release/3.2.x
